src/vm-repair/azext_vm_repair/_help.py

```diff
@@ -0,0 +1,40 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from knack.help_files import helps
+
+helps['vm repair'] = """
+    type: group
+    short-summary: Auto repair commands to fix faulty VMs. Recommended to use all commands with the --verbose flag.
+    long-summary: >
```

Comment:
long-summary:
When Azure Virtual Machines are faulty, usually they be can't connected using RDP or SSH. To troubleshoot and correct them, the machines will be deleted and the OS disks will be attached to another rescue machine to perform repair. VM repair scripts will enable Azure users to self-repair faulty VMs by running simple commands within the Azure Portal cloud shell.

Reply:
I know I used it first but is the word 'faulty' the best here? And the machines will not be deleted. The OS disk will be copied then attached to another newly created rescue machine.